ember/app/abilities/identity.js

```diff
@@ -4,20 +4,24 @@ export default class IdentityAbility extends BaseAbility {
   get canList() {
     return this.isStaff;
   }
+
   get canAdd() {
     return this.isStaff;
   }
+
+  canEdit() {
+    return this.isStaffOrOwnIdentity(this.model);
+  }
+
   get canEditOrganisation() {
-    return this.isStaff;
+    return this.isStaff || this.model;
```

Comment:
Well technically I can 
```js
get abilityModel() {
    return this.args.profileEditRoute
      ? this.membership?.firstObject.identity
      : this.changeset.data;
  }
```
But I don't get how this will check for the `edit organisation identity` ability.

Reply:
Alright. As the passed `this.model` (AbilityModel) can only be fetched by an authorized user for this org, it also validates the ability to edit the org. Got it :+1: